shell/edgex-publish-swagger.sh

```diff
@@ -9,19 +9,24 @@ if [ -z "$ARCH" ] || [ "$ARCH" != "arm64" ] ; then
     # NOTE: APIKEY needs to be a pointer to a file with the key. This will need to be set locally from your environment or from Jenkins
     APIKEY_VALUE=`cat $APIKEY`
 
-    API_VERSION=${API_VERSION:-2.0.0}
-    API_FOLDER=${API_FOLDER:-v2}
+    # API_VERSION=${API_VERSION:-2.0.0}
```

Comment:
Is there a reason to leave these lines (12, 13, 22) commented out? If they're not necessary, they should be removed. If you're not sure if they should be removed, this should be tested more thoroughly. Keep in mind that removed lines will be kept in git history, in the case that they need to be restored later.

Reply:
@MightyNerdEric Thanks for your advice. I removed the lines commented out.